### PR TITLE
DF-2072 - Add Unit Tests

### PR DESCRIPTION
**Pull request by karahanb**
# Description

This PR includes the following proposed change(s):

- Adds unit tests for DF-2072, GET Documents List endpoint in DocumentsApiDelegate.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Ran JUnit Tests locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: digitalforms-api/src/test/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DocumentsApiControllerTests.java [hunk 1]
```diff
@@ -323,4 +361,133 @@ public void documentsAssociationNoticeDocumentIdPostMissingNoticeTypeCdFailure()
...
+    	String correlationId = DigitalFormsConstants.UNIT_TEST_CORRELATION_ID;
+    	String noticeNo = DigitalFormsConstants.UNIT_TEST_NOTICE_NUMBER;
+    	Long impoundmentId = DigitalFormsConstants.UNIT_TEST_IMPOUNDMENT_ID;
+		
+		// Mock underlying VIPS REST Search 
+		vipsSearch.setRespMsg(DigitalFormsConstants.DIGITALFORMS_SUCCESS_MSG);
+		vipsSearch.setRespCd(DigitalFormsConstants.VIPSWS_SUCCESS_CD);
+        Mockito.when(service.searchImpoundment(correlationId, noticeNo)).thenReturn(vipsSearch);
+        
+        // Mock underlying VIPS REST Get Documents response returning no documents in the response list even though the
```
Comment (shaunmillargov) [inline digitalforms-api/src/test/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DocumentsApiControllerTests.java:432]: Hi Burak. This comment is confusing. A list of document may not exist even if we have a impoundment Id or prohibition Id returned from the prior search. This is not a BAD vips case. Maybe just remove the comment. Thanks.

**Review by shaunmillargov — CHANGES_REQUESTED** [review]
I've requested one little change please Burak. Easy one. Just a comment 
Other than that, looks good. Thanks.

File: digitalforms-api/src/test/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DocumentsApiControllerTests.java (hunk 1 above)
Comment (karahanb) [reply digitalforms-api/src/test/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DocumentsApiControllerTests.java:432]: Sorry that's a copy paste error. I will remove that part of the comment.

**Review by shaunmillargov — APPROVED** [review]
Thanks for the update Burak. Looks good.

**Pull request merged**